data_validation/combiner.py

```diff
@@ -210,7 +209,11 @@ def _calculate_differences(
 
 def _pivot_result(result, join_on_fields, validations, result_type):
     all_fields = frozenset(result.schema().names)
-    validation_fields = all_fields - frozenset(join_on_fields)
+    validation_fields = (
+        all_fields - frozenset(join_on_fields)
+        if "hash__all" not in join_on_fields
```

Comment:
Instead of checking if `hash__all` exists, we should have a flag that indicates custom query column versus row i.e.`validate custom-query row` or `validate custom-query column`
This way someone can't provide a `--sum` and a `--hash` in the same validation which would break

Reply:
Done.